src/constants.tsx

```diff
@@ -6,3 +6,5 @@ export type SET_FILTER = typeof SET_FILTER;
 
 export const SET_NETWORK = 'SET_NETWORK';
 export type SET_NETWORK = typeof SET_NETWORK;
+
+export const mainUrl = 'https://conseil-staging.cryptonomic-infra.tech/tezos/';
```

Comment:
Could we please add a file called `config.json` in the `src` directory and read the URLs and API keys from there? There should be entries for both `betanet` and `zeronet` and, depending on which one is selected, the right URL and API key should be used.